nodeadm/internal/containerd/oci_config.go

```diff
@@ -0,0 +1,148 @@
+package containerd
+
+import (
+	"errors"
+	"io/fs"
+	"os/exec"
+	"reflect"
+	"slices"
+	"strings"
+
+	"github.com/awslabs/amazon-eks-ami/nodeadm/internal/api"
+	"github.com/awslabs/amazon-eks-ami/nodeadm/internal/util"
+	"github.com/pelletier/go-toml/v2"
+	"go.uber.org/zap"
+)
+
+type instanceTypeMixin struct {
+	instanceFamilies []string
+	apply            func(*[]byte) error
```

Comment:
slices are passed by reference (for the most part), so a `*[]byte` can just be `[]byte` in this case

Reply:
okay, will change it